package.json

```
@@ -8,6 +8,7 @@
     "@capacitor/ios": "^2.4.6",
     "@ionic/react": "^5.0.7",
     "@ionic/react-router": "^5.0.7",
+    "@ionic/storage": "^3.0.6",
```

Este es un comentario de prueba:)
```suggestion
    "@ionic/storage": "^3.0.6",Una sugerencia de cambio
```